index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <h1>John Doe</h1>
+
+            <div id="navig">
+                <label for="humberger" id="hum">&#9776;</label>
+                <input type="checkbox" id="humberger"/>
+                <div id="navitems">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+                    <a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a>
+                </div>
+                </div>
```

Comment:
- [ ] The hamburger button is a button and should be inside a proper `<button>` tag for better semantics. Kindly change that.
 - [ ] The hamburger button should be inside a `<nav>` tag. Kindly change that for better semantics.
 - [OPTIONAL] The mobile menu functionality will be implemented in future projects. If you want to start doing it, please try to follow the visual from the Figma template.